inference-engine/tests/functional/plugin/shared/include/behavior/infer_request_config.hpp

```diff
@@ -42,10 +42,8 @@ TEST_P(InferConfigTests, canSetExclusiveAsyncRequests) {
         ASSERT_NO_THROW(ie->SetConfig(config, targetDevice));
     }
     // Load CNNNetwork to target plugins
-    if (targetDevice.find(CommonTestUtils::DEVICE_AUTO) == std::string::npos) {
-        auto execNet = ie->LoadNetwork(cnnNet, targetDevice, config);
-        execNet.CreateInferRequest();
-    }
+    auto execNet = ie->LoadNetwork(cnnNet, targetDevice, config);
+    execNet.CreateInferRequest();
```

Comment:
BTW, although not related directly to this PR, but I see no diff in the code of this test and the test below (withoutExclusiveAsyncRequests),
I think the **below** test should set the EXCLUSIVE_ASYNC_REQUESTS to NO (as it name implies).
@apankratovantonp and @ilya-lavrenov fyi

Reply:
@ilya-lavrenov pls comment, so @coneypo can fix the test if it is really incorrect (as his PR touches these tests)

Reply:
@myshevts I agree, should be fixed

Reply:
I suggest to change the second (duplicated) test to set EXCLUSIVE_ASYNC_REQUESTS to NO and remove the checks on the #executors.